user-service/user-domain/src/main/kotlin/com/grepp/quizy/domain/user/UserService.kt

```diff
@@ -0,0 +1,22 @@
+package com.grepp.quizy.domain.user
+
+import org.springframework.stereotype.Service
+
+@Service
+class UserService (
+    private val userAppender: UserAppender,
+    private val userReader: UserReader,
+    private val userRemover: UserRemover
+) {
+    fun appendUser(user: User) : User {
+        return userAppender.appendUser(user)
+    }
```

Comment:
저도 가끔 실수하긴 하는데 type 일 시에는 `: 반환타입` 상속일 때는 ` : 인터페이스명`이 컨벤션이라고 합니다.

```suggestion
    fun appendUser(user: User): User {
        return userAppender.appendUser(user)
    }
```

Reply:
넵 지적해주셔서 감사합니다!